### ISSUE TITLE
HW 2 Feedback

### ISSUE THREAD

**Issue by alexchaomander** [zfcousens/DAT_SF_12#3]
- With the new ipython notebook, some of the plot margins are messed up. This happened to my notebooks as well. Make sure to double check them in the future!
- Good use of scale. Adding the additional scaling jumps the predictive accuracy from 70% to 90%. Exploration of your data through plots and other means would have revealed this.
- There are other ways to calculate the accuracy of your model. For instance most models will have a score function. i.e. "clf.score(X_test, y_test)" http://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html
- But what you wrote essentially does that albeit in a more manual way. It's good practice to write out the entire calculation so that you know the internals behind functions.
- Nice comments after each snippet of code / plot. It helps not just the reader but yourself when you go back and work on the functions. 
- All in all, good work!
